types/trace/trace.go

```diff
@@ -42,8 +42,9 @@ type Event struct {
 	Syscall               string       `json:"syscall"`
 	StackAddresses        []uint64     `json:"stackAddresses"`
 	ContextFlags          ContextFlags `json:"contextFlags"`
-	EntityID              uint32       `json:"-"`    // task unique identifier (*)
-	Args                  []Argument   `json:"args"` // args are ordered according their appearance in the original event
+	ThreadEntityId        uint32       `json:"threadEntityId"`  // thread task unique identifier (*)
+	ProcessEntityId       uint32       `json:"processEntityId"` // process unique identifier (*)
```

Comment:
I'm +1 to have parentEntityId. I have planned to have the 2 from the beginning (process and parent). Then after our discussions, the thread group leader appeared and now for me the 3 should walk together. I bet there will be a future need to access parent and we won't have parent start_time to calculate the hash.